types/utils/todo.d.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare global {
+  export type TODO = any;
+  export type CombinedAuditAndAuditRef = TODO;
+  export type LighthouseLastRunDetail = TODO;
+  export type LighthouseScore = TODO;
+  export type TemplateResult = TODO;
```

Comment:
Lit has types. Add a TODO comment saying that we should import them?

Reply:
I created a definition file that will expose `TemplateResult` to our JS definitions without the need of importing it from the `node_modules` folder.